lib/telephone_number/test_data_generator.rb

```diff
@@ -0,0 +1,91 @@
+module TelephoneNumber
+  require 'nokogiri'
+  require 'httparty'
+  require 'yaml'
+
+  class TestDataGenerator
```

Comment:
I'd be interested in implementing a "delta" feature as well so that we reduce the risk of someone getting pissed and blacklisting our IP